dm/ctl/simulator/bw_list.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simulator
+
+import (
+	"github.com/pingcap/dm/dm/config"
+	"github.com/pingcap/dm/dm/ctl/common"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb-tools/pkg/filter"
+	"github.com/spf13/cobra"
+)
+
+type bwListResult struct {
+	Result         bool                `json:"result"`
+	Msg            string              `json:"msg"`
+	DoTables       map[string][]string `json:"do-tables,omitempty"`
+	IgnoreTables   map[string][]string `json:"ignore-tables,omitempty"`
+	WillBeFiltered string              `json:"will-be-filtered,omitempty"`
+}
+
+// NewBWListCmd creates a BWList command
+func NewBWListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bw-list [-w worker] [-T table-name] <config-file>",
+		Short: "check the black-white-list info for tables",
+		Run:   bwListFunc,
+	}
+	cmd.Flags().StringP("table-name", "T", "", "the table name we want to check for the black white list")
```

Comment:
1. Is `table` better than `table-name`?
2. Can we specify this flag multi times? (`-T tbl1 -T tbl2`)